File: apps/python/app-01-ecommerce-catalog/services/catalog-service/src/routes/auth_routes.py
from flask import Blueprint

from src.controllers import auth_controller


auth_bp = Blueprint("auth", __name__, url_prefix="/api/auth")
auth_bp.add_url_rule("/login", view_func=auth_controller.login, methods=["POST"])
auth_bp.add_url_rule("/logout", view_func=auth_controller.logout, methods=["POST"])
auth_bp.add_url_rule("/me", view_func=auth_controller.get_me, methods=["GET"])

File: apps/typescript/app-14-telemedicine/src/routes/authRoutes.ts
import { Router } from "express";
import { AuthController } from "../controllers/AuthController";

export function createAuthRoutes(controller: AuthController) {
  const router = Router();
  router.post("/register", controller.register);
  router.post("/login", controller.login);
  router.post("/logout", controller.logout);
  router.get("/me", controller.me);
  return router;
}
